features/polyExplorer/src/components/clusterStory/clusterStory.css

```diff
@@ -54,15 +54,16 @@
 
 .cluster-story .big-first-letter::first-letter {
     float: left;
-    font-size: 60px;
+    font-family: "Jost Bold";
```

Comment:
Is this change of size intended?

Reply:
Yes, it is intended! Figma says 45px here at this state, I don't know why someone put 60 there, but I know that this kind of capital and also its font-family was changed by design-team recently.

Reply:
I would suspect 60 non-bold might look similar to 45 :D